pkg/apis/v1beta1/ec2nodeclass.go

```diff
@@ -329,6 +329,12 @@ type EC2NodeClass struct {
 	Status EC2NodeClassStatus `json:"status,omitempty"`
 }
 
+// We need to bump the EC2NodeClassHashVersion under these condition when we make an update to the EC2NodeClass CRD
+// 1. A  adds default values to existing fields that are included in the hash calculations
```

Comment:
We should have whatever comment that we have here match the comment that we have for the version over in the kubernetes-sigs/karpenter repo